src/tests/tests/heuristic.js

```diff
@@ -146,20 +146,4 @@ QUnit.test("Cookie attributes shouldn't add to entropy", (assert) => {
   }
 });
 
-QUnit.test("CloudFlare cookies should get ignored", (assert) => {
```

Comment:
Yes, done!

Reply:
Tangential, but I wonder if it'd make sense to also add [`cf_clearance`](https://support.cloudflare.com/hc/en-us/articles/200170156-Understanding-the-Cloudflare-Cookies) to the exclusion list. Does it currently cause issues?
After doing a quick search it seems like it was briefly mentioned here: https://github.com/EFForg/privacybadger/issues/1237.

Reply:
Thank you for the updates! I should take a look next week.

`cf_clearance`: I am not sure, not that I noticed so far. What is the workflow to get this cookie? Is there a live site I could review this on?

Reply:
We set the `cf_clearance` cookie when the client successfully solves one of the interstitial challenges (non-interactive challenge or something like a CAPTCHA). Example:

<img width="1139" alt="challenge" src="https://user-images.githubusercontent.com/9093796/118202554-1ce4b280-b452-11eb-9e77-435f50a87b3b.png">

You can get one of these here:
- Non-interactive: https://migueldemoura.com/js-challenge
- Interactive (CAPTCHA): https://migueldemoura.com/captcha

The cookie is used to avoid presenting more challenges to the client on subsequent requests. Not sure if relevant, but the cookie is scoped to the domain the user is accessing (for the previously examples it'd be `.migueldemoura.com`).